macros/plot_scurves_by_thresh.py

```diff
@@ -1,4 +1,4 @@
-from plot_scurve import *
+from macros.plot_scurve import *
```

Comment:
Fixed; and added the imports I assumed it was trying to take from plot_scurve.py (ROOT and OptionsParser).